converse.js

```diff
@@ -3009,7 +3010,7 @@
             getRoomJIDAndNick: function (nick) {
                 nick = nick || this.model.get('nick');
                 var room = this.model.get('jid');
-                var node = Strophe.escapeNode(Strophe.getNodeFromJid(room));
+                var node = Strophe.getNodeFromJid(room);
```

Comment:
I think it gets escape twice because when the room is created the JID is escaped the first time.